src/modules/rover_differential/module.yaml

```diff
@@ -120,3 +112,59 @@ parameters:
         increment: 0.01
         decimal: 2
         default: 0.5
+
+      RD_MAX_SPEED:
+        description:
+          short: Maximum speed
+          long: Maximum speed the rover can drive.
+        type: float
+        unit: m/s
+        min: 0
+        max: 100
+        increment: 0.01
+        decimal: 2
+        default: 7
+
+      RD_MAX_YAW_RATE:
+        description:
+          short: Maximum yaw rate for the rover.
+        type: float
+        unit: deg/s
+        min: 0.01
+        max: 1000
+        increment: 0.01
+        decimal: 2
+        default: 90
+
+      RD_MISS_VEL_DEF:
+        description:
+          short: Default rover velocity during a mission
+        type: float
+        unit: m/s
+        min: 0
+        max: 100
+        increment: 0.01
+        decimal: 2
+        default: 1
+
+      RD_TRANS_TRN_DRV:
+        description:
+            short: Heading error threshhold to switch from spot turning to driving
+        type: float
+        unit: rad
+        min: 0.001
+        max: 180
+        increment: 0.01
+        decimal: 3
+        default: 0.0872665
+
+      RD_TRANS_DRV_TRN:
```

Comment:
I wonder if those really need to be two different params. Could you also work with a hysteresis but with the same threshold for both directions?

Reply:
Not dismissing this as a valid option, but I usually tuned it s.t. the transition to go from driving to turning is more than double the transition from turning to driving. My idea is that you only want to come to a stop and do a spot turn if the heading error is quite large but once you come to a stop you should take the time to get the orientation as good as possible before continuing, therfore needing a lower threshold.